### PR TITLE
St victor goloschapov lesson 3 level 1

### PR DESCRIPTION
**Pull request by VictorGoloschapov**
Good evening,

Please review finished tasks from level 1.
Thank you.

### CODE REVIEW

File: src/main/java/students/student_Victor_Goloschapov/lesson_3/homework/level_1/task_1/Robot.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package lesson_3.homework.level_1.task_1;
+
+class Robot {
+    void sayHello() {
```
Comment (antdolg1) [inline src/main/java/students/student_Victor_Goloschapov/lesson_3/homework/level_1/task_1/Robot.java:4]: please add access modifier - public
`
public void sayHello() { ...`

**Review by antdolg1 — COMMENTED** [review]
Formatting is on point, package structure is organised and easy to understand. Overall great job. Only one remark, please add access modifier - `public` before `sayHello()` and `sayYourName()` methods.

File: src/main/java/students/student_Victor_Goloschapov/lesson_3/homework/level_1/task_1/Robot.java (hunk 1 above)
Comment (VictorGoloschapov) [reply src/main/java/students/student_Victor_Goloschapov/lesson_3/homework/level_1/task_1/Robot.java]: I have added public modifier to methods sayHello() and sayYourName() in every task

**Review by antdolg1 — APPROVED** [review]
Good job, keep on going 👍

**Pull request merged**